### PR TITLE
Added integration for Photo Editor app with new screen manager SDK

### PR DESCRIPTION
**Pull request by soalb-m**
Added integration for Photo Editor app with new screen manager SDK

### CODE REVIEW

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Sorin! :-)

Please, fix the ktlint issues and you are fine to go.

File: PhotoEditor/app/src/androidTest/java/com/microsoft/device/display/samples/photoeditor/PhotoEditorUITest.kt [hunk 1]
```diff
@@ -34,6 +43,23 @@ import org.hamcrest.CoreMatchers.`is` as iz
 class PhotoEditorUITest {
     @get:Rule
     val activityRule = ActivityTestRule(MainActivity::class.java)
+    private var screenInfoListener = ScreenInfoListenerImpl()
+
+    @Before
+    fun setup() {
+        val screenManager = ScreenManagerProvider.getScreenManager()
+        screenManager.addScreenInfoListener(screenInfoListener)
+//        activityRule.launchActivity(null)
```
Comment (ancirja-m) [inline PhotoEditor/app/src/androidTest/java/com/microsoft/device/display/samples/photoeditor/PhotoEditorUITest.kt:52]: Remove commented code from everywhere

Comment (CesarValiente) [inline PhotoEditor/app/src/androidTest/java/com/microsoft/device/display/samples/photoeditor/PhotoEditorUITest.kt:52]: Is this line not needed anymore? if not, please delete it.

File: PhotoEditor/app/src/androidTest/java/com/microsoft/device/display/samples/photoeditor/PhotoEditorUITest.kt [hunk 2]
```diff
@@ -34,6 +43,23 @@ import org.hamcrest.CoreMatchers.`is` as iz
...
+    }
+
+    @After
+    fun tearDown() {
+        val screenManager = ScreenManagerProvider.getScreenManager()
+        screenManager.removeScreenInfoListener(screenInfoListener)
+        screenInfoListener.resetScreenInfo()
+        screenInfoListener.resetScreenInfoCounter()
+//        activityRule.finishActivity()
```
Comment (CesarValiente) [inline PhotoEditor/app/src/androidTest/java/com/microsoft/device/display/samples/photoeditor/PhotoEditorUITest.kt:61]: Is this line not needed anymore? if not, please delete it.

File: PhotoEditor/app/src/main/java/com/microsoft/device/display/samples/photoeditor/MainActivity.kt [hunk 3]
```diff
@@ -86,17 +88,17 @@ class MainActivity : AppCompatActivity(), ScreenInfoListener {
         val setupLayoutRunnable = Runnable {
             setupLayout()
 
-            val image = findViewById<ImageFilterView>(R.id.image)
+//            val image = findViewById<ImageFilterView>(R.id.image)
```
Comment (CesarValiente) [inline PhotoEditor/app/src/main/java/com/microsoft/device/display/samples/photoeditor/MainActivity.kt:91]: We can remove this line if not needed anymore

File: PhotoEditor/app/src/main/java/com/microsoft/device/display/samples/photoeditor/MainActivity.kt [hunk 4]
```diff
@@ -86,17 +88,17 @@ class MainActivity : AppCompatActivity(), ScreenInfoListener {
...
 
             // Restore image
-            val vm by viewModels<PhotoEditorVM>()
-            vm.getImage().value?.let {
-                image.setImageDrawable(vm.getImage().value)
+//            val viewModel by viewModels<PhotoEditorVM>()
```
Comment (CesarValiente) [inline PhotoEditor/app/src/main/java/com/microsoft/device/display/samples/photoeditor/MainActivity.kt:94]: We can remove this line if not needed anymore

File: PhotoEditor/app/src/main/java/com/microsoft/device/display/samples/photoeditor/MainActivity.kt [hunk 5]
```diff
@@ -33,62 +33,102 @@ import androidx.constraintlayout.utils.widget.ImageFilterView
...
+    }
+
+    override fun onPause() {
+        super.onPause()
+        ScreenManagerProvider.getScreenManager().removeScreenInfoListener(this)
+    }
 
-        // Default property value for ImageFilterView attributes (state of original image)
-        private const val ORIGINAL_STATE = 1f
+    override fun onConfigurationChanged(newConfig: Configuration) {
```
Comment (ancirja-m) [inline PhotoEditor/app/src/main/java/com/microsoft/device/display/samples/photoeditor/MainActivity.kt]: Please remove this

**Review by ancirja-m — APPROVED** [review]

**Review by CristianVerdes — APPROVED** [review]

**Pull request merged**